management/user.go

```diff
@@ -724,3 +749,22 @@ func (m *UserManager) DeleteAllAuthenticationMethods(ctx context.Context, userID
 	err = m.management.Request(ctx, "DELETE", m.management.URI("users", userID, "authentication-methods"), nil, opts...)
 	return
 }
+
+// GetRefreshTokens retrieves details for a user's refresh tokens.
+//
+// It allows pagination using the provided options. For more information on pagination, refer to:
+// https://pkg.go.dev/github.com/auth0/go-auth0/management#hdr-Page_Based_Pagination
+//
+// See: https://auth0.com/docs/api/management/v2#!/Users/get-refresh-tokens-for-user
+func (m *UserManager) GetRefreshTokens(ctx context.Context, userID string, opts ...RequestOption) (r *RefreshTokenList, err error) {
```

Comment:
```suggestion
func (m *UserManager) ListRefreshTokens(ctx context.Context, userID string, opts ...RequestOption) (r *RefreshTokenList, err error) {
```

Should we call this `ListRefreshTokens` so it's similar to the pre-established convention throughout the SDK?